src/index.scss

```diff
@@ -35,6 +34,81 @@ code {
   display: none !important;
 }
 
+// buttons
+.button-blue-fill, .button-blue-outline, .button-red-outline, .button-grey-fill {
+  border-radius: 5px;
+  transition-duration: 0.1s;
+  transition-timing-function: ease-out;
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.button-blue-fill, .button-grey-fill {
+  border: none;
+}
+
+.button-blue-outline, .button-red-outline {
+  background: white;
+  border-style: solid;
+  border-width: 2px;
+}
+
+.button-blue-fill {
+  color: white;
+  background-color: #54a0f1;
+  &:hover {
+    background-color: #7bb8f9;
+  }
+}
+
+.button-blue-outline {
+  color: #54a0f1;
+  border-color: #54a0f1;
+  &:hover {
+    color: #7bb8f9;
+    border-color: #7bb8f9;
+  }
+}
+
+.button-red-outline {
+  color: #e95f5f;
+  border-color: #e95f5f;
+  &:hover {
+    color: #ee9999;
+    border-color: #ee9999;
+  }
+}
+
+.button-grey-fill {
+  color: #2b2b2b;
+  background-color: #c4c4c4;
+  &:hover {
+    background-color: #747474;
+  }
+}
+
+
+.save-button {
+  margin-bottom: 20px;
```

Comment:
i have a feeling we actually want to remove the margin-bottom: 20px from buttons, but we can test that more